components/scheduler/src/Scheduler.svelte

```diff
@@ -77,7 +73,7 @@
       "Schedule",
     );
     event_title = getPropertyValue(
-      internalProps.event_title || editorManifest.event_title,
+      internalProps.event_title,
```

Comment:
All these changes are rendered unnecessary with the recent updates to the manifest store re: external_manifest_ids, right? If so, I think we also have a `show_hosts` property doing things the old way in `Availability.svelte`